examples/error-type-binding-pattern/error_type_binding_pattern.bal

```diff
@@ -1,37 +1,33 @@
 import ballerina/io;
 
-type SampleError error<string, map<anydata|error>>;
+type SampleErrorData record {
+    string detail;
```

Comment:
Let's change this field to something else. A `detail` field in a detail mapping could get confusing.